src/main/java/tornadofx/Layouts.kt

```diff
@@ -28,6 +32,38 @@ fun GridPane.row(title: String? = null, op: (Pane.() -> Unit)? = null) {
     addRow(properties[GridPaneRowIdKey] as Int, *fake.children.toTypedArray())
 }
 
+fun GridPane.constraintsForColumn(columnIndex: Int): ColumnConstraints {
```

Comment:
The `ObservableList<ColumnConstraints>` returned by [`getColumnConstraints`](http://docs.oracle.com/javafx/2/api/javafx/scene/layout/GridPane.html#getColumnConstraints()) is usually empty. The indices in that list correspond with the columns, so the third column from the left will look for `getColumnConstraints()[2]`. If the list is too short, the gridpane just [follows its internal defaults](http://hg.openjdk.java.net/openjfx/8/master/rt/file/tip/modules/graphics/src/main/java/javafx/scene/layout/GridPane.java#l1141)

So, this function populates the list just enough so that there's a `ColumnConstraints` object at the correct position in the array which it can return.